src/components/Icon/Icon.test.tsx

```diff
@@ -28,7 +28,7 @@ test.each([
   ["accent", "c_text.accent"],
   ["danger", "c_text.danger"],
   ["warning", "c_text.warning"],
-] as const)('applies the correct class for tone="%s"', (tone, className) => {
+] as const)("tone 값에 따라 class가 올바르게 적용됨", (tone, className) => {
   const { container } = render(<Basic tone={tone} muted={false} />);
```

Comment:
```suggestion
] as const)('tone "%s"값에 따라 class가 올바르게 적용됨', (tone, className) => {
  const { container } = render(<Basic tone={tone} muted={false} />);
```

https://vitest.dev/api/#test-each
오 리뷰하면서 알게된건데 %s의 순서에 따라 배열의 값을 가져올수 있네요?

Reply:
@Sunjae95 네! 오.. 제가 한국어로 변경하면서 간과했던 부분 말씀주셔서 감사합니다.
반영해서 merge하겠습니다!